zebedee-cms/src/main/java/com/github/onsdigital/zebedee/ZebedeeConfiguration.java

```diff
@@ -163,7 +173,7 @@ public ZebedeeConfiguration(Path rootPath, boolean enableVerificationAgent) thro
             datasetClient = new DatasetAPIClient(
                     Configuration.getDatasetAPIURL(),
                     Configuration.getDatasetAPIAuthToken(),
-                    Configuration.getServiceAuthToken());
+                    getServiceAuthToken());
```

Comment:
Why is this now a static import but the two lines above aren't? Looks a little messy here now, I'd suggest making them all a static import or none.